src/backend/cdb/cdbllize.c

```diff
@@ -659,6 +659,19 @@ ParallelizeCorrelatedSubPlanMutator(Node *node, ParallelizeCorrelatedPlanWalkerC
 		return ParallelizeCorrelatedSubPlanMutator(node, ctx);
 	}
 
+	if (IsA(node, ModifyTable))
+	{
+		Plan   *mplan = (Plan *) node;
```

Comment:
pgindent sugests more ident, see 655 line

Reply:
done

Reply:
are you apply pgindent to your changes?

Reply:
Yep, that's it